src/Components/Main/Pages/MessagesPage/MessageForm/MessageFormContainer.js

```diff
@@ -0,0 +1,29 @@
+import { Formik } from 'formik';
+import * as Yup from 'yup';
+import { MessageForm } from './MessageForm';
+
+const validationSchema = Yup.object().shape({
+  text: Yup.string()
+    .max(100, 'Must be shorter than 100 characters')
+    .required('Required')
+});
+
+export const MessageFormContainer = ({addMessage}) => {
+  const onSubmit = (formData, {setSubmitting, resetForm}) => {
+    setSubmitting(false);
```

Comment:
Is it correct to 
```
 setSubmitting(false);
```
When we submit it. Should we set  setSubmitting(true); here? =)

Reply:
I used setSubmitting(false) because I specified the attribute disabled={isSubmitting} in the button, and setSubmitting sets as true by default.
Redundant logic for such case is corrected.